pkg/operator/ceph/object/user/controller.go

```diff
@@ -393,7 +396,6 @@ func generateUserConfig(user *cephv1.CephObjectStoreUser) admin.User {
 	userConfig := admin.User{
 		ID:          user.Name,
 		DisplayName: displayName,
-		Keys:        make([]admin.UserKeySpec, 1),
```

Comment:
why not initialize this?

Reply:
@travisn: The memory will be allocated when keys are added. If we initialised Key with length. the `GetUser()` api from go-ceph interrupt key is passed to the request and it fails [here](https://github.com/ceph/go-ceph/blob/master/rgw/admin/user.go#L110)